doc/features/components/core/statements/batch/README.md

```diff
@@ -0,0 +1,29 @@
+# Batch statements
+
+Use [BatchStatement] to execute a set of queries as an atomic operation (refer to [Batching inserts, updates and deletes][batch_dse] to understand how to use batching effectively):
+
+```csharp
+PreparedStatement preparedInsertExpense =
+    session.Prepare(
+        "INSERT INTO cyclist_expenses (cyclist_name, expense_id, amount, description, paid) "
+            + "VALUES (:name, :id, :amount, :description, :paid)");
+SimpleStatement simpleInsertBalance =
+    new SimpleStatement(
+        "INSERT INTO cyclist_expenses (cyclist_name, balance) VALUES (?, 0) IF NOT EXISTS",
+        "Vera ADRIAN");
```

Comment:
I would suggest using query markers for all parameters.
```suggestion
    new SimpleStatement(
        "INSERT INTO cyclist_expenses (cyclist_name, balance) VALUES (?, ?) IF NOT EXISTS",
        "Vera ADRIAN", 0);
```

Reply:
done!